specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -16,6 +16,7 @@
       - [Folder Structure](#folder-structure)
       - [Package Publishing](#package-publishing)
   - [Detailed Design](#detailed-design)
+    - [Build/Test Configuration](#buildtest-configuration)
     - [Front-End Components](#front-end-components)
```

Comment:
May pull front-end components before build/test config, since closer to abstraction and design than the practical configurations.